fmc/src/flow/rt_alias.rs

```diff
@@ -226,23 +230,32 @@ impl RtAliasLayer {
 
         // Sign the the `To Be Signed` portion
         cprintln!(
-            "[art] Signing Cert with AUTHO
+            "[alias rt] Signing Cert with AUTHO
             RITY.KEYID = {}",
             auth_priv_key as u8
         );
 
+        // Sign the AliasRt To Be Signed DER Blob with AliasFMC Private Key in Key Vault Slot 7
+        // AliasRtTbsDigest = sha384_digest(AliasRtTbs) AliaRtTbsCertSig = ecc384_sign(KvSlot5, AliasFmcTbsDigest)
+
         let sig = Crypto::ecdsa384_sign(env, auth_priv_key, tbs.tbs());
         let sig = okref(&sig)?;
+        // Clear the authority private key
+        cprintln!(
+            "[alias rt] Erasing AUTHORITY.KEYID = {}",
+            auth_priv_key as u8
+        );
+        env.key_vault.erase_key(auth_priv_key)?;
```

Comment:
Should it be erased or locked? On impactless update reset I would expect FMC to need to use this key again and I'm not sure if ROM will re-derive it.

Reply:
I was following what ROM is doing in the FMC Alias flow. : https://github.com/chipsalliance/caliptra-sw/blob/main/rom/dev/src/flow/cold_reset/fmc_alias.rs#L176

I am going to lock instead.